packages/vats/src/nameHub.js

```diff
@@ -100,6 +117,10 @@ export const makeNameHubKit = () => {
       );
       nameAdmin.update(key, newValue, adminValue);
     },
+    onUpdate: fn => {
+      assert(!fn || !updateCallback);
```

Comment:
took me a second to interpret these two cases. please include an assertion message. consider a type annotation

Reply:
restructured a bit and added assertion message.

the type is already annotated, though somewhat indirectly via `NameAdmin` typedef in `vats/src/types.js`.